home/static/home/hero.css

```
@@ -17,6 +17,12 @@ html, body {
   line-height: 1.5;
 }
 
+.hero-bg {
```

Como essa classe é nossa, podemos chamá-la de apenas `background`.

```suggestion
.background {
```